### PR TITLE
Add positional arguments to create_account and new_project

### PR DESCRIPTION
**Pull request by potatodepaulo**
Resolves https://github.com/nearprotocol/near-shell/issues/46

### CODE REVIEW

File: get-config.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
 module.exports = function getConfig() {
-    const configPath = process.cwd() + '/src/config';
+    const configPath = process.cwd() + '/src/config.js';
```
Comment (vgrichina) [inline get-config.js]: hmm, why did you have to change this?

File: index.js [hunk 2]
```diff
@@ -35,18 +35,18 @@ exports.clean = async function() {
 // Only works for dev environments
 exports.createDevAccount = async function(options) {
     const keyPair = await KeyPair.fromRandomSeed();
-
+    let accountId = options.accountId;
```
Comment (vgrichina) [inline index.js]: nit: `const { accountId } = options;` (same tricks can be used to extract other options)

File: test/new_project.sh [hunk 3]
```diff
@@ -1,10 +1,22 @@
...
-../bin/near new_project
+../../bin/near new_project
 npm install
 npm uninstall near-shell
-npm install ../
+npm install ../../
 NODE_ENV=development npm run test
+cd ..
+rm  -rf tmp-project
+# test generating new project in new dir
```
Comment (vgrichina) [inline test/new_project.sh]: full test cycle is pretty slow, I think it's enough to test option with explicit directory full cycle and just check that both options write out `package.json`

File: index.js (hunk 2 above)
Comment (potatodepaulo) [reply index.js]: Ah yeah, good call. Will add.

File: test/new_project.sh (hunk 3 above)
Comment (potatodepaulo) [reply test/new_project.sh]: Can run tests in the first cycle and will just check the package.json moves in the second.

File: get-config.js (hunk 1 above)
Comment (potatodepaulo) [reply get-config.js]: Wasn't finding the config.js file locally. Worked when I changed it. 🤷‍♀

File: test/new_project.sh [hunk 4]
```diff
@@ -16,7 +20,9 @@ rm  -rf tmp-project
...
-npm install
-npm uninstall near-shell
-npm install ../../
-NODE_ENV=development npm run test
\ No newline at end of file
+FILE=package.json
+if test -f "$FILE"; then
+  echo "${GREEN}$FILE exists. Have a cookie!"
+else
+  echo "${RED}ERROR: file not found."
```
Comment (vgrichina) [inline test/new_project.sh]: need to add `exit 1` or it won't return error code (and so won't fail CI)

Comment (potatodepaulo) [reply test/new_project.sh]: done

**Pull request merged**